cvmfs/swissknife_gc.cc

```diff
@@ -169,13 +173,21 @@ int CommandGc::Main(const ArgumentList &args) {
     }
   }
 
+  StatisticsDatabase *stats_db = StatisticsDatabase::OpenStandardDB(repo_name);
```

Comment:
This should always succeed, shouldn't it?  We could crash if `stats_db == NULL` and then wouldn't need to test later on.  Same for the other commands.